org.eclipse.jdt.ls.tests/.gitignore

```diff
@@ -0,0 +1,3 @@
+# temp files generated when running test projects
```

Comment:
this should not be necessary, this indicates the tests are changing source directories, which is bad.

Reply:
so we want to keep the .gitignore unchanged, as this is our canary telling us the tests are bad

Reply:
OK. I just let the canary go this time 😄